### PR TITLE
PS-9074 8.0.36-28 release notes and release note tasks (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
new file:   docs/release-notes/8.0.36-28.md
	modified:   docs/release-notes/release-notes_index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.0.36-28.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Get started with [Quickstart Guide for Percona Server for MySQL](../quickstart-overview.md).
+
+[Percona Server for MySQL 8.0.36-28](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[MySQL 8.0.36 [Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-36.html) and enterprise-grade features developed by Percona.
+
+## Release highlights
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.36 and included in Percona Server for MySQL are the following:'
+
+* OpenSSL is now version 3.0.12.
```
Comment (percona-ysorokin) [inline docs/release-notes/8.0.36-28.md]: The issue is that we almost always use the OpenSSL libraries from the OS.
Even Oracle says it more carefully
```
For platforms on which OpenSSL libraries are bundled, the linked OpenSSL library for MySQL Server has been updated to version 3.0.12
```
So, I would simply remove this item from the highlights.

File: docs/release-notes/8.0.36-28.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+* Updated GnuPG build key (A8D3785C). This key is used to sign MySQL downloadable packages. The previous key has expired. For details, see [Signature checking using GNUPG].
```
Comment (percona-ysorokin) [inline docs/release-notes/8.0.36-28.md]: This is irrelevant to Percona products. Percona packages use completely different set of keys for package signing.
Please, remove.

Comment (percona-ysorokin) [reply docs/release-notes/8.0.36-28.md]: Instead, I would put
```
The hashing algorithm employed yielded poor performance when using a HASH field to check for uniqueness. (Bug #109548, Bug #34959356)
```
as a noticeable improvement.

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

**Review by percona-ysorokin — APPROVED** [review]
LGTM

File: docs/release-notes/8.0.36-28.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+* The hashing algorithm employed yielded poor performance when using a HASH field to check for uniqueness. (Bug #109548, Bug #34959356)
+
+* All [statement instrument elements] that begin with `statement/sp/%`, except `statement/sp/stmt`, are disabled by default.
+
+Find the complete list of bug fixes and changes in the [MySQL 8.0.36 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-36.html).
+
+
+##  Bug fixes
+
+[PS-9018]: The replica stalls when `log_slave_updates=0`and a DDL is executed against a non-replicated database on that replica.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.0.36-28.md]: `log_slave_updates=0`and -> add a space

`log_slave_updates=0`and a DDL is -> are

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**